doc/rados/operations/pools.rst

```diff
@@ -25,7 +25,10 @@ Pools provide:
   resources. When setting up multiple pools, set an appropriate number of PGs
   for each pool and for the cluster as a whole. Each PG belongs to a specific
   pool: when multiple pools use the same OSDs, make sure that the **sum** of PG
-  replicas per OSD is in the desired PG-per-OSD target range. 
+  replicas per OSD is in the desired PG-per-OSD target range. See :ref:`Setting
+  the Number of Placement Groups <setting the number of placement groups>` for
```

Comment:
The link does not work for me.
And the only target I can see is 'setting number ...' (i.e. - without the 'the'

Reply:
@ronen-fr:

I have no trouble with the link. Have a look at the rendered document and let me know if you can't follow the link:

https://ceph--55918.org.readthedocs.build/en/55918/rados/operations/pools/#pools

Reply:
Thanks. Indeed it works.

Reply:
Qapla.